src/components/forging/forgingStats.js

```diff
@@ -41,9 +42,13 @@ class ForgingStats extends React.Component {
               <CardText>
               <div className={grid['col-xs-12']}>
                 <div className={`${grid.row}  ${grid['between-xs']}`}>
-                  <span className='title'> {cardObj.label} </span>
+                  <span className='title'>
+                    { cardObj.label === 'Last 24 hours' ?
+                      this.props.t(cardObj.label) :
+                      this.props.t('Last x days', { day: cardObj.label }) }
```

Comment:
This condition is ugly and hacky. I would move the definition of `statCardObjects` inside `ForgingStats.render` so that it can use `this.props.t`and here the code can stay the way it was.

Reply:
The key should be `Last {{day}} days` for better readability. Or maybe `Last {{count}} days` as well as the value.